be/Cf.Domain/Aggregates/Reviews/Review.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Cf.Domain.Aggregates.Jobs;
+
+namespace Cf.Domain.Aggregates.Reviews;
+
+public class Review : Entity
```

Comment:
I think we would want to also have author's of review info like name atleast.

Reply:
Added